addons/mail/static/src/components/channel_invitation_form_selectable_partner/channel_invitation_form_selectable_partner.xml

```diff
@@ -2,24 +2,24 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ChannelInvitationFormSelectablePartner" owl="1">
-        <div class="o_ChannelInvitationFormSelectablePartner d-flex align-items-center px-3 py-1 btn-light" t-on-click="() => channelInvitationForm.onClickSelectablePartner(selectablePartner)" t-att-data-partner-id="selectablePartner.id" t-attf-class="{{ className }}" t-ref="root">
+        <div class="o_ChannelInvitationFormSelectablePartner d-flex align-items-center px-3 py-1 btn-light" t-on-click="() => channelInvitationFormSelectablePartnerView.channelInvitationFormOwner.onClickSelectablePartner(channelInvitationFormSelectablePartnerView.partner)" t-att-data-partner-id="channelInvitationFormSelectablePartnerView.partner.id" t-attf-class="{{ className }}" t-ref="root">
```

Comment:
Shouldn't `channelInvitationFormSelectablePartnerView` be guarded?

Reply:
Was needed at some point, but it's no longer needed actually